Fix Bilbao subway icon

matheusgomesms opened the pull request:
Since the last PRs to add Bilbao icons (#4211 and #4338), the correct names of station and entrances were not added, I suppose. This PR aims to fix this. Not sure if I added everything that is needed (adding a new subway icon is a bit cumbersome, it seems), hope it's good!

**PS.: Strings must be regenerated. Could you please do that? Thanks! :D**

CC @endim8 @starsep 

Signed-off-by: Matheus Gomes

<img src="https://user-images.githubusercontent.com/86851490/218894773-12c8398c-fd46-4f66-9e2a-f1540042eae5.jpeg" width=50% height=05%>

starsep commented on data/types.txt:
```diff
@@ -1363,6 +1363,7 @@ amenity|recycling|container
 *railway|station|subway|bangkok
 *railway|station|subway|beijing
 *railway|station|subway|bengalore
+*railway|station|subway|bilbao
```
I think new type shouldn't be added in the middle - number of line here is id of type.
Maps from previous release wouldn't be backward compatible

AFAIK it's ok to replace `mapswithme` or add at the end

vng commented:
You should add bilbao subway entrance type into mapcss-mapping.csv only.
Let say instead of current ```deprecated|deprecated;1502;x``` entry

classificator.txt, types.txt will be generated based on this csv file with tools/unix/generate_drules.sh tool

matheusgomesms commented:
Thanks for all the info! I reverted those files, and added the last two lines on mapcss-mapping.csv.

Also, I kept the additions of data/strings/types_strings.txt, I wasn't sure if I had to add those lines or not.

vng reviewed (commented):
UPD: Sorry, everything is ok.
